src/components/Header.js

```diff
@@ -0,0 +1,22 @@
+import {
+  faCircleArrowLeft,
+  faGear,
+  faMicrophone,
+} from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { Link } from 'react-router-dom';
+
+const Heading = () => (
+  <nav className="nav_bar">
+    <Link to="/">
+      <FontAwesomeIcon icon={faCircleArrowLeft} id="arrow-left" />
+    </Link>
```

Comment:
- [ ] Please add a conditional rendering to this `arrow-left` button  so that it will only appear on the details page 😃 👍

Reply:
The Return arrow is on both pages. 
![image](https://user-images.githubusercontent.com/100999396/205465020-9e38dbdd-d53d-4432-9ebb-48f417a4312e.png)